web/src/pages/barcoding.js

```diff
@@ -1,9 +1,335 @@
-import React from 'react'
-
-export default function BarcodingPage() {
-    return (
-        <div>
-            <h1>this is the Barcoding Page</h1>
-        </div>
-    )
+import React from 'react';
+import tw from 'twin.macro';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+import styled from 'styled-components';
+
+import Layout from '../components/Layout'
+
+const HeroStyle = styled.div`
+  width: 100vw;
+  ${tw`w-screen`};
+  top: -160px;
+  margin-bottom: 200px;
+
+  .gatsby-image-wrapper {
+    top: 100px
+
+  }
+  @media screen and (max-width: 480px) {
+   .gatsby-image-wrapper {
+     width: 100vw;
+     top: 150px
+   }
+   margin-bottom: 200px;
+ }
+//  @media screen and (max-width: 700px) and (min-width: 400px) {
+//   .gatsby-image-wrapper {
+//     width: 100vw;
+//   }
+//   ${tw`w-screen`};
+// }
```

Comment:
if this isn't being used I think we should remove it. I know there's code comments all over this project but I'd like to clean that up moving forward.